data/field_config.go

```diff
@@ -182,6 +182,18 @@ type ExplorePanelsState any
 type TimeRange struct {
 	From time.Time `json:"from,omitempty"`
 	To   time.Time `json:"to,omitempty"`
+	// Use RawTimeRangeTime or RawTimeRangeString when passing it to frontend
+	Raw interface{} `json:"raw,omitempty"`
```

Comment:
Is this only required when doing relative time?

Reply:
Yep, totally agree. I wasn't sure if there's a way to create a union type in Go to match what we have in frontend. But I think it makes sense to make it string-only 🤔 (it can be converted to a `DateTime` in frontend when needed)